wofi-pass

```diff
@@ -101,7 +103,26 @@ function usage() {
 EOF
 }
 
+get_config_file () {
+    [ ! "$XDG_CONFIG_HOME" ] && export XDG_CONFIG_HOME="$HOME/.config"
+    configs=("$XDG_CONFIG_HOME/wofi-pass/config"
+             "/etc/wofi-pass.conf")
+
+    # return the first config file with a valid path
+    for config in "${configs[@]}"; do
+        # '-n' is needed in case WOFI_PASS_CONFIG is not set
```

Comment:
What is `WOFI_PASS_CONFIG`?
Either you add it to `config_files` or delete this comment with `-n` query in the next line.

Reply:
Apologies, this is the remnant of the ENV variable controlled config path that doesn't work with `set -u`. I'll remove this